CoreScriptsRoot/Modules/PlayerDropDown.lua

```diff
@@ -227,7 +234,7 @@ spawn(function()
 end)
 
 local function isBlocked(userId)
-	if (BlockedList[userId] ~= nil and BlockedList[userId] == true) then
+	if (BlockedList[userId]) then
```

Comment:
BlockedList should now be a dictionary of user ids, structured like

key        value
userid    true

Since it only depends on the userId being a key, it doesn't matter what the value is.